src/core/namespace.c

```diff
@@ -1762,11 +1762,11 @@ static int apply_one_mount(
                         (void) mkdir_parents(mount_entry_path(m), 0755);
 
                         q = make_mount_point_inode_from_path(what, mount_entry_path(m), 0755);
-                        if (q < 0) {
-                                if (q != -EEXIST) // FIXME: this shouldn't be logged at LOG_WARNING, but be bubbled up, and logged there to avoid duplicate logging
-                                        log_warning_errno(q, "Failed to create destination mount point node '%s', ignoring: %m",
-                                                          mount_entry_path(m));
-                        } else
+                        if (q < 0 && q != -EEXIST)
```

Comment:
Hmm, `EEXIST` could also mean that the target inode typo mismatches what we shall create. Maybe, make `make_mount_point_inode_from_path` return -EALREADY or so to reflect that case, and use it here?

Reply:
If that happens the next mount will fail anyway, so it wouldn't really make any difference. This needs to be backported, so the less churn the better

Reply:
OK.